src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/browser-base/browser-base.component.ts

```diff
@@ -195,26 +196,29 @@ export abstract class BaseBrowserComponent<TEntity extends IEntityBase> extends
     /**
      * Click event to trigger selection (ctrl + shift click)
      */
-    public itemClicked($event: MouseEvent, index: number): void {
+//   [TEMPLATE CLEANSING] Type changed to any as MouseEvent | Event, type Event alway gave error
+    public itemClicked(event: any, index: number): void {
```

Comment:
Create different implementation for different event types

Reply:
What about this?